homeassistant/components/vallox/sensor.py

```diff
@@ -123,15 +131,17 @@ class ValloxSensorEntityDescription(SensorEntityDescription):
     """Describes Vallox sensor entity."""
 
     metric_key: str | None = None
-    sensor_type: type[ValloxSensor] = ValloxSensor
+    entity_type: type[ValloxSensorEntity] = ValloxSensorEntity
+    round_ndigits: int | None = None
 
 
-SENSORS: tuple[ValloxSensorEntityDescription, ...] = (
+SENSOR_ENTITIES: tuple[ValloxSensorEntityDescription, ...] = (
     ValloxSensorEntityDescription(
         key="current_profile",
         name="Current profile",
         icon="mdi:gauge",
-        sensor_type=ValloxProfileSensor,
+        entity_type=ValloxProfileSensor,
+        entity_registry_enabled_default=False,
```

Comment:
Ahh. I moved it to "select" entity in the full PR https://github.com/home-assistant/core/pull/75431.
So this would be a read-only duplicate for that "select" not to introduce a breaking change for people.

Need to keep entity_registry_enabled_default True for this PR if we are not taking "select" entity in for now.